pkg/container-suseconnect/configuration_test.go

```diff
@@ -29,10 +29,10 @@ func TestGetLocationPath(t *testing.T) {
 
 	strs := []string{
 		"does/not/exist",
-		"data/products-sle12.json",
+		"../../test/products-sle12.json",
```

Comment:
Why don't you move this `data` directory into this internal package? Moreover, this directory only contains fixtures which are relevant only to internal tests, no?

Reply:
Yep, I moved the data into the internal `testdata` folder.